RELEASING.md

```diff
@@ -1,18 +1,25 @@
 
 ### To push a release version
 
-1. Run `utilities/update_pom_version.sh` from the repository's base directory.
-This script takes an optional argument denoting the new version.  By default, if the current version is X.Y.Z-SNAPSHOT, the script will update the version in all the pom.xml files to X.Y.Z.  If desired, another version can be supplied via command line argument instead.
+1. Make sure the team agrees that it is time to release. Verify that all unit and integration tests for the last commit have passed.
 
-2. Create a PR to update the pom.xml version.  If releasing a new client library, this PR should also update javadoc grouping in the base directory's [pom.xml](./pom.xml).
-PRs that don't release new modules should look something like [#225](https://github.com/GoogleCloudPlatform/google-cloud-java/pull/225).  PRs that do release a new module should also add the appropriate packages to the javadoc groups "SPI" and "Test helpers", as shown in [#802](https://github.com/GoogleCloudPlatform/google-cloud-java/pull/802) for `google-cloud-dns`. After this PR is merged into GoogleCloudPlatform/google-cloud-java, Travis CI will push a new website to GoogleCloudPlatform/gh-pages, push a new artifact to the Maven Central Repository, and update versions in the README files.  Do not merge in any non-release-related pull requests between the start of step 2 and the end of step 6.  Between these steps, the project version is a non-snapshot version, so any commits to the master branch will cause Travis to spend extra resources attempting to redeploy artifacts.
+2. Run `utilities/update_pom_version.sh` from the repository's base directory.
+This script takes an optional argument denoting the new version. By default, if the current version is X.Y.Z-SNAPSHOT, the script will update the version in all the pom.xml files to X.Y.Z. If desired, another version can be supplied via command line argument instead. Commit this version locally:
 
-3. Before moving on, verify that the artifacts have successfully been pushed to the Maven Central Repository.  Open Travis CI, click the ["Build History" tab](https://travis-ci.org/GoogleCloudPlatform/google-cloud-java/builds), and open the second build's logs for Step 2's PR.  Be sure that you are not opening the "Pull Request" build logs.  When the build finishes, scroll to the end of the log and verify that the artifacts were successfully staged and deployed.  Search for `google-cloud` on the [Sonatype website](https://oss.sonatype.org/#nexus-search;quick~google-cloud) and check the latest version number.  In rare cases (when the Maven plugin that determines the version of the repository fails), the artifacts may not be deployed even if the version in the pom.xml files doesn't contain `SNAPSHOT`.  If the artifacts weren't deployed due to invalid version parsing or a flaky test, rerun the build.
+  ```
+  git commit -m "Release [VERSION HERE]"
```

Comment:
It seems always the case that we would create a commit everytime we update the version. Why not make `utilities/update_pom_version.sh` does the git commit automatically?